content/references/xrpljs2-migration-guide.md

```diff
@@ -227,36 +253,36 @@ In ripple-lib 1.x all methods and properties were on instances of the `RippleAPI
+| `requestNextPage()` | [`Client.requestNextPage()`](https://js.xrpl.org/classes/Client.html#requestNextPage) | |
+| `computeBinaryTransactionHash()` | [`xrpl.hashes.hashTx()`](https://js.xrpl.org/modules.html#hashes) | |
+| `classicAddressToXAddress()` | [`xrpl.classicAddressToXAddress()`](https://js.xrpl.org/modules.html#classicAddressToXAddress) | Now a static method on the module. |
+| `xAddressToClassicAddress()` | [`xrpl.xAddressToClassicAddress()`](https://js.xrpl.org/modules.html#xAddressToClassicAddress) | Now a static method on the module. |
 | `renameCounterpartyToIssuer(object)` | (Removed - see Notes column) | No longer needed because xrpl.js always uses `issuer` already. |
 | `formatBidsAndAsks()` | (Removed - see Notes column) | No longer needed after changes to `getOrderbook()`. |
-| `connect()` | `Client.connect()` | |
-| `disconnect()` | `Client.disconnect()` | |
-| `isConnected()` | `Client.isConnected()` | |
+| `connect()` | [`Client.connect()`](https://js.xrpl.org/classes/Client.html#connect) | |
+| `disconnect()` | [`Client.disconnect()`](https://js.xrpl.org/classes/Client.html#disconnect) | |
+| `isConnected()` | [`Client.isConnected()`](https://js.xrpl.org/classes/Client.html#isConnected) | |
 | `getServerInfo()` | (Removed - see Notes column) | Use [`Client.request()`](https://js.xrpl.org/classes/Client.html#request) to call the [server_info method][] instead. |
-| `getFee()` | (Removed - see Notes column) | Use `Client.autofill()` to provide a sensible [transaction cost][] automatically, or use `Client.request({"command": "fee"})` to look up information about the current transaction cost (in _drops of XRP_). |
-| `getLedgerVersion()` | `Client.getLedgerIndex()` | |
-| `getTransaction()` | `Client.request()` | Use [`Client.request()`](https://js.xrpl.org/classes/Client.html#request) to call the [tx method][] instead. **Warning:** Unlike `getTransaction()`, the `tx` method can return [results that are not validated and final](#validated-results). Be sure to look for `"validated": true` in the response object before taking action in response to a transaction. |
+| `getFee()` | (Removed - see Notes column) | Use [`Client.autofill()`](https://js.xrpl.org/classes/Client.html#autofill) to provide a sensible [transaction cost][] automatically, or use `Client.request({"command": "fee"})` to look up information about the current transaction cost (in _drops of XRP_). |
+| `getLedgerVersion()` | [`Client.getLedgerIndex()`](https://js.xrpl.org/classes/Client.html#getLedgerIndex) | |
+| `getTransaction()` | [`Client.request()`](https://js.xrpl.org/classes/Client.html#request) | Use [`Client.request()`](https://js.xrpl.org/classes/Client.html#request) to call the [tx method][] instead. **Warning:** Unlike `getTransaction()`, the `tx` method can return [results that are not validated and final](#validated-results). Be sure to look for `"validated": true` in the response object before taking action in response to a transaction. |
```

Comment:
Is `[tx method][]` supposed to have a link? (Context "to call the [tx method][] instead. **Warning:**..."

Reply:
This is one of the Markdown reference links. Standard markdown syntax allows for something like this:

```
The [tx method][] is a link.

[tx method]: https://xrpl.org/tx.html
```

Which produces this:

> The [tx method][] is a link.
>
> [tx method]: https://xrpl.org/tx.html

In our case, we have a single file that provides the link definitions for a bunch of things including all the API methods and gets included using an `{% include '_snippets/rippled-api-links.md' %}` macro at the end of the file. The macro is one of Dactyl's extensions on standard Markdown, so the GitHub preview doesn't show it correctly, but it works when you actually build the site.